### PR TITLE
Added CGR cinemas brand

### PR DESCRIPTION
**Pull request by QuentinS33**
Added CGR cinemas which operates mainly in France

### CODE REVIEW

File: brands/amenity/cinema.json [hunk 1]
```diff
@@ -80,6 +80,17 @@
       "short_name": "Century"
     }
   },
+  "amenity/cinema|CGR": {
+    "countryCodes": ["fr"],
+    "matchNames": ["cgr"],
```
Comment (bhousel) [inline brands/amenity/cinema.json]: This line can go away, matching is case-insensitive.
```suggestion
```

File: brands/amenity/cinema.json [hunk 2]
```diff
@@ -80,6 +80,17 @@
...
+    "tags": {
+      "amenity": "cinema",
+      "brand": "CGR",
+      "brand:wikidata": "Q2931406",
+      "brand:wikipedia": "fr:Circuit_Georges-Raymond",
```
Comment (bhousel) [inline brands/amenity/cinema.json]: Wiki tags need to have underscore `_` replaced with space ` `..
```suggestion
      "brand:wikipedia": "fr:Circuit Georges-Raymond",
```

File: brands/amenity/cinema.json (hunk 1 above)
Comment (QuentinS33) [reply brands/amenity/cinema.json]: Ok, I didn't know the case-insensitive for matching brands. I will correct this.

File: brands/amenity/cinema.json (hunk 2 above)
Comment (QuentinS33) [reply brands/amenity/cinema.json]: Ok, i didn't know that rule. I copied the brand wikipedia from URL. I will correct this.

**Comment by bhousel** [comment]
Thanks @QuentinS33 looks great!  and welcome 👋

**Pull request merged**